### PR TITLE
Expose `@ember/test-waiters` in addition to `ember-test-waiters` module names

### PR DESCRIPTION
**Pull request by scalvert**
Module rename as part of [RFC 0581](https://github.com/emberjs/rfcs/blob/master/text/0581-new-test-waiters.md)

### CODE REVIEW

File: addon/ember-test-waiters/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from '@ember/test-waiters';
```
Comment (rwjblue) [inline addon/ember-test-waiters/index.js:1]: We should add an issue to add a deprecation here I think.

File: addon/@ember/test-waiters/build-waiter.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Primitive, TestWaiter, TestWaiterDebugInfo, WaiterName } from 'ember-test-waiters';
+import { Primitive, TestWaiter, TestWaiterDebugInfo, WaiterName } from '@ember/test-waiters';
```
Comment (rwjblue) [inline addon/@ember/test-waiters/build-waiter.ts:1]: Can you update this to be a relative path?

File: index.js [hunk 3]
```diff
@@ -10,4 +10,8 @@ module.exports = {
 
     forceHighlander(this.project);
   },
+
+  treeForAddon(tree) {
+    return this.processedAddonJsFiles(tree);
+  },
```
Comment (rwjblue) [inline index.js:16]: Can you add a comment RE: not calling super (and not supporting templates or styles) and explaining what we are actually doing here?

File: addon/ember-test-waiters/index.js (hunk 1 above)
Comment (scalvert) [reply addon/ember-test-waiters/index.js:1]: Agree.

File: index.js (hunk 3 above)
Comment (scalvert) [reply index.js:16]: 👍

File: addon/@ember/test-waiters/build-waiter.ts (hunk 2 above)
Comment (scalvert) [reply addon/@ember/test-waiters/build-waiter.ts:1]: Yep.

**Comment by scalvert** [comment]
@rwjblue added a sanity check task that imports from `ember-test-waiters` to ensure the legacy module import works.

**Comment by rwjblue** [comment]
@scalvert - Changes seem good, should we also change the package name while we are at it?

**Comment by scalvert** [comment]
I removed the deprecation and updated some other references to the old module name. I also extended the highlander check to include the new module name.

Can you take another look, @rwjblue?

File: README.md [hunk 4]
```diff
@@ -45,12 +45,12 @@ This addon implements the design specified in [RFC 581](https://github.com/ember
 ## Installation
 
 ```
-ember install ember-test-waiters
+ember install @ember/test-waiters
```
Comment (rwjblue) [inline README.md:48]: I don't think this part will work just yet, probably should revert back:

```suggestion
ember install ember-test-waiters
```

File: package.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ember-test-waiters",
+  "name": "@ember/test-waiters",
```
Comment (rwjblue) [inline package.json:2]:
```suggestion
  "name": "ember-test-waiters",
```

**Pull request merged**